conbench/tests/entities/test_history.py

```diff
@@ -72,7 +72,7 @@ def assert_equal_leeway(comparison, reference, figs=4):
     ("mean", "closest_defaultbranch_ancestor"): [
         None,
         None,
-        28.48,
+        None,
```

Comment:
one outcome of this: now contenders with errors won't have z-scores (which was always the intention anyway)